README.md

```diff
@@ -1,30 +1,23 @@
-#WSO2 API Management Server
+# WSO2 API Management
 
+WSO2 API Management is an Open Source API Management tool that allows APIs/Services to be Secured, Monitored and Rate Controlled. This tool allows API Developers to design, pubilsh, version and manage API lifecycles. It allows application developers to discover and start consuming APIs. The lightweight API Gateway intercepts all API requests applying Security, Rate Limiting and Monitoring aspects on your APIs.
 
+More information about this product can be found [here](http://wso2.com/api-management/)
 
-Running Samples
----------------
+## Building from source
 
-Sample Integration Flow configurations are available at samples/SequenceDiagramDSLSamples directory.
+1. Get a clone or download the source from this repository (https://github.com/wso2/carbon-apimgt)
+2. Run the Maven command ``mvn clean  install`` from within the carbon-apimgt directory.
+3. Get a clone or download the source from this repository (https://github.com/wso2/product-apim)
+4. Run the Maven command ``mvn clean install`` from within the product-apim directory.
 
-Configuration can be deployed to server by dropping the file to <CARBON_HOME>/deployment/integration-flows/ directory.
+## Running the product
 
-
-####Sample Configuration
-
-```sh
-@Path ("/stock")
-@Source (protocol="http", host="localhost", port=8080)
-@Api (tags = {"stock_info","stock_update"}, description = "Rest api for do operations on admin", produces = MediaType.APPLICATION_JSON)
-package com.sample;
-
-constant endpoint stockEP = new HTTPEndPoint("http://localhost:8081/stockquote/WSO2");
-
-@GET
-@PUT
-@POST
-@Path ("/passthrough")
-resource passthrough (message m) {
-   reply invoke(stockEP, m);
-}
-```
+1. Download the WSO2 Identity Server from [here](http://wso2.com/identity-and-access-management#download)
+2. Extract the downloaded wso2is-5.3.0.zip file
+3. Go to the wso2is-5.3.0/bin directory and execute the wso2server.sh script using the command ``./wso2server.sh start``
+4. Find the API Manager product distribution from product-apim/product/target
+5. Extract the distribution. Ex: unzip wso2apim-3.0.0.zip
```

Comment:
Let's leave it as 3.0.0.zip, I put it as an example so that we don't have to be explicit.